src/projects/republic/config/englishRepublicLabels.ts

```diff
@@ -34,4 +34,14 @@ export const englishRepublicLabels = Object.assign({}, englishLabels, {
 
   // text panel titles:
   self: "Text",
+
+  COM: "commission",
+  HOE: "capacity",
+  LOC: "location",
+  ORG: "organisation",
+  PER: "person",
+
+  SEARCH_CATEGORY: "Search for ",
```

Comment:
Is this extra space necessary?

Reply:
Fixed in [79b00cd](https://github.com/knaw-huc/textannoviz/pull/132/commits/79b00cddf073c6c21706527641caaaca4ee3e9b9)